plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -173,9 +178,9 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
         parent
       );
 
-      if (plugin._config.requestHook) {
+      if (config.requestHook) {
```

Comment:
Suggestion: check that this is a function ```typeof config.requestHook === "function"``` (probably really should create a helper for this check at some point).

Why?: This is to protect us from being passed faulty config and causing the app to throw/crash.

Reply:
We've got the `safeExecuteInTheMiddle` protecting us here when calling the `requestHook`
Anyhow - this is not really part of this PR - i only extracted the config to a variable... 
We can surely fix this - just not part of this PR.
Thanks!